nms/app/packages/magmalte/app/views/tracing/TracingDashboard.js

```diff
@@ -0,0 +1,187 @@
+        ]}
+      />
+      <TracingTable />
+    </>
+  );
+}
+
+type TracingRowType = {
+  traceID: string,
+  state: 'COMPLETED' | 'ACTIVE',
+  gatewayID: string,
+  traceType: string,
+};
+
+function TracingTableRaw(_: WithAlert) {
+  const [currRow, setCurrRow] = useState<TracingRowType>({});
+  const classes = useStyles();
+  const ctx = useContext(TraceContext);
+  const {networkId} = useContext(NetworkContext);
+  const enqueueSnackbar = useEnqueueSnackbar();
+  const traceMap = ctx.state;
+  const tableData = tracesToRows(traceMap);
+  const tableColumns = [
+    {title: 'Trace ID', field: 'traceID'},
+    {title: 'State', field: 'state'},
+    {title: 'Gateway ID', field: 'gatewayID'},
+    {title: 'Trace Type', field: 'traceType'},
+  ];
+
+  const TraceFilter = () => {
+    return <CreateTraceButton />;
+  };
+
+  const handleStop = async () => {
+    if (currRow.state === 'COMPLETED') {
+      enqueueSnackbar('Call trace ' + currRow.traceID + ' already stopped.', {
+        variant: 'error',
+      });
+      return;
+    }
+
+    try {
+      // $FlowFixMe[prop-missing]: Suppress type error, cannot refine type
+      await ctx.setState?.(currRow.traceID, {
+        requested_end: true,
+      });
+      enqueueSnackbar('Call trace ended successfully', {
+        variant: 'success',
+      });
+    } catch (e) {
+      const errMsg = e.response?.data?.message ?? e.message ?? e;
+      enqueueSnackbar('Failed stopping call trace: ' + errMsg, {
+        variant: 'error',
+      });
+    }
+  };
+
+  const handleDownload = async () => {
+    if (currRow.state != 'COMPLETED') {
+      enqueueSnackbar('Call trace ' + currRow.traceID + ' is still active', {
+        variant: 'error',
+      });
+      return;
+    }
+
+    if (networkId) {
+      // TODO(andreilee): Build download link based on generated API bindings
+      window.location.href =
+        '/nms/apicontroller/magma/v1/networks/' +
+        networkId +
+        '/tracing/' +
+        currRow.traceID +
+        '/download';
+    }
+  };
+
+  return (
+    <div className={classes.dashboardRoot}>
+      <Grid container spacing={4}>
+        <Grid item xs={12}>
+          <CardTitleRow
+            icon={HistoryIcon}
+            label={'Call Traces'}
+            filter={TraceFilter}
+          />
+          <div>
+            <ActionTable
+              data={tableData}
+              columns={tableColumns}
+              handleCurrRow={(row: TracingRowType) => setCurrRow(row)}
+              menuItems={[
+                {
+                  name: 'Download',
+                  // $FlowFixMe[incompatible-type]
+                  handleFunc: handleDownload,
+                },
+                {
+                  name: 'Stop',
+                  // $FlowFixMe[incompatible-type]
+                  handleFunc: handleStop,
+                },
+              ]}
+              options={{
+                actionsColumnIndex: -1,
+                pageSizeOptions: [10, 20],
```

Comment:
You can set the default page size to 10, by adding pageSize here
https://material-table.com/#/docs/all-props